ecl/hql/hqlgram.hpp

```diff
@@ -677,9 +677,9 @@ class HqlGram : implements IErrorReceiver, public CInterface
     void enterVirtualScope();
     void leaveScope(const attribute & errpos);
     IHqlExpression * leaveLamdaExpression(attribute * modifierattr, attribute & exprattr);
-    IHqlScope * closeLeaveScope(const YYSTYPE & errpos);
+    IHqlScope * closeLeaveScope(const ECLYYSTYPE & errpos);
```

Comment:
Could replace all references to ECLYYTYPE  with attribute (since it is a #defined).  It would look cleaner and it is currently very inconsistent.